lib/iris/analysis/maths.py

```diff
@@ -384,8 +384,8 @@ def exponentiate(cube, exponent, in_place=False):
     """
     _assert_is_cube(cube)
 
-    def power(data, out=None):
-        return np.power(data, exponent, out)
+    def power(data):
+        return np.power(data, exponent)
```

Comment:
Actually no - the tests were suggesting that I had preserved lazy data when exponentiating a cube, which was the full extent of the change I was aiming for here.

However I seem to have hit a bit of the worst of both worlds with this closure nonetheless: no use of biggus and no inplace possibility.